src/containers/CreateSecret/CreateSecret.js

```diff
@@ -388,7 +388,8 @@ function mapStateToProps(state) {
 const mapDispatchToProps = {
   createSecret,
   fetchServiceAccounts,
-  patchSecret
+  patchSecret,
+  selectNamespace
```

Comment:
missing import for the `selectNamespace` action

Reply:
Fixed (not sure how things worked...but they did/do without) 👀